components/raftstore/Cargo.toml

```diff
@@ -62,6 +62,7 @@ prost = { version = "0.6", optional = true }
 protobuf = "2.8"
 quick-error = "1.2.3"
 raft = { version = "0.6.0-alpha", default-features = false }
+raft-proto = { git = "https://github.com/pingcap/raft-rs", branch = "master", default-features = false }
```

Comment:
You can use the protos by `raft::eraftpb`.

Reply:
I tried, but failed, seem `ConfChangeI` is not included by `raft::eraftpb`.

Reply:
Indeed.